pandas/core/sparse/series.py

```diff
@@ -581,98 +580,13 @@ def combine_first(self, other):
         return dense_combined.to_sparse(fill_value=self.fill_value)
 
     def to_coo(self, row_levels=(0, ), column_levels=(1, ), sort_labels=False):
-        """
-        Create a scipy.sparse.coo_matrix from a SparseSeries with MultiIndex.
```

Comment:
Can you keep those docstrings here (or share it with the delegate method)

Reply:
My hope is to deprecate SparseSeries entirely for 0.24.0. But I suppose we'll want a docstring for the deprecation period, so I'll use shared docs.